global_planner/cfg/GlobalPlanner.cfg

```diff
@@ -26,9 +26,13 @@ orientation_enum = gen.enum([
               "Negative y axis points along the path, except for the goal orientation"),
     gen.const("Bidirectional", int_t, 7,
               "Forward or backward depending on what requires less turning from start to path and from path to goal"),
+    gen.const("Omni", int_t, 8,
```

Comment:
S: rename to `FixedOrientation` ? Also, the description would be more clear IMO if it said sth like
`"Keep the same orientation as the orientation at the start pose, except at the goal"`